src/web/components/ReaderRevenueLinks.tsx

```diff
@@ -117,13 +133,190 @@ const subMessageStyles = css`
 	margin: 5px 0;
 `;
 
-export const ReaderRevenueLinks: React.FC<Props> = ({
+interface Cta {
+	url: string;
+	text: string;
+}
+interface SupportHeaderProps {
+	content: {
+		heading: string;
+		subheading: string;
+		primaryCta?: Cta;
+		secondaryCta?: Cta;
+	};
+	tracking: TestMeta;
+}
+interface SupportHeaderData {
+	module: {
+		url: string;
+		name: string;
+		props: SupportHeaderProps;
+	};
+	meta: TestMeta;
+}
+
+const ReaderRevenueLinksRemote: React.FC<{
```

Comment:
This is the component used by the variant in the AB test. It renders the remotely-fetched module